ci/ci_test.sh

```diff
@@ -17,6 +17,13 @@ elif [[ $SELECTED_TEST_SUITE == $"all" ]]; then
   rake test:plugins         # Run all plugins tests
 else
   echo "Running core tests"
-  rake test:install-core    # Install core dependencies for testing.
+  rake plugin:install-core    # Install core dependencies for testing.
```

Comment:
because I need to bundle the new logstash-devutils. so I can do just a bootstrap. test:install-core is  doing some stuff beforehand.